src/app/pages/wallets/my-wallets/wallet-form-modal/domains.wallet-modal.types.ts

```diff
@@ -0,0 +1,3 @@
+export interface IWalletModalData {
```

Comment:
You forgot to rename this file :) It should be `pages-wallets-my-wallets-wallet-form-modal.types.ts` and should be in `wallet-form-modal` directory.